kubernetes/iac/environments/local/README.md

```diff
@@ -0,0 +1,8 @@
+# iac-environments-local
+
+Apply the cluster-resources module to your local development cluster
+
+### Port-Forward the APISIX Dashboard or Gateway
+> kubectl port-forward [name of the apisix-dashboard-pod] 9000:9000
+
+> kubectl port-forward [name of the apisix-gateway-pod] 9080:9080
```

Comment:
Can you add a section that describes that user/password is admin/admin? Also please add a follow up issue to use a newly generated k8s secret instead.